packages/apps/job-launcher/server/src/modules/job/job.service.ts

```diff
@@ -1460,14 +1458,17 @@ export class JobService {
   }
 
   private async getOracleFee(
-    oracleAddress: string,
     chainId: ChainId,
+    oracleAddress?: string,
```

Comment:
Dont make this param optional, it is not intuitive at all how to use this method